### PR TITLE
Make the package independent of heppy

### PR DESCRIPTION
**Pull request by JavierCVilla**
- Replace every `from heppy.FCChhAnalyses.` to `from FCChhAnalyses.`
- Add a `setup.py` to install FCChhAnalyses as a python package
- Extend gitignore file

### CODE REVIEW

File: FCChhAnalyses/FCChh/tth_mumu/analysis.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
 # pre-produced input files
 comp = cfg.Component(
     'example',
-     files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_ll012j_5f/events100.root"]
-     files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_tth01j_5f_hmumu/events0.root"]
+     files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_ll012j_5f/events100.root"],
+     #files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_tth01j_5f_hmumu/events0.root"]
```
Comment (JavierCVilla) [inline FCChhAnalyses/FCChh/tth_mumu/analysis.py]: @clementhelsens this needs review, the current code does not work as it is, so we could exclude one of the two files or do the following:

```python
     files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_ll012j_5f/events100.root",
              "root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_tth01j_5f_hmumu/events0.root"
               ]
```

**Comment by JavierCVilla** [comment]
Need to modified the `README.md`

File: FCChhAnalyses/FCChh/tth_mumu/analysis.py (hunk 1 above)
Comment (vvolkl) [reply FCChhAnalyses/FCChh/tth_mumu/analysis.py]: Hi Javier, your proposed change is fine. `comp` is a way to manually test the analysis with a small / local sample. In fact maybe you can comment l.36 where it says 
```
# uncomment to try with local file 
selectedComponents = [comp]
```

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                'FCChhAnalyses.FCChh.vbs_ww',
+                'FCChhAnalyses.HELHC',
+                'FCChhAnalyses.HELHC.Dijet_reso',
+                'FCChhAnalyses.HELHC.RSGraviton_ww',
+                'FCChhAnalyses.HELHC.Zprime_ll',
+                'FCChhAnalyses.HELHC.Zprime_mumu_flav_ano',
+                'FCChhAnalyses.HELHC.Zprime_tautau',
+                'FCChhAnalyses.HELHC.Zprime_tt',
+                'FCChhAnalyses.HELHC.tth_boosted',
+                ]
```
Comment (vvolkl) [inline setup.py]: proposed change:
```

#!/usr/bin/env python2
  
from setuptools import setup, find_packages
import glob

setup(name='FCChhAnalyses',
      version='1.0',
      description='Produce flat ROOT trees using FCCSW EDM in heppy',
      author='Clement Helsens',
      author_email='clement.helsens@cern.ch',
      url='https://github.com/HEP-FCC/FCChhAnalyses',
      requires=['heppy'], # heppyfwk if installed with pip
      packages=find_packages(),
      package_dir={"FCChhAnalyses": "../FCChhAnalyses"},
      scripts=glob.glob('scripts/*')
     )
```

**Comment by clementhelsens** [comment]
that looks good to me, thanks

**Pull request merged**